lib/collection/ssh-auth.js

```diff
@@ -0,0 +1,152 @@
+var _ = require('../util').lodash,
+    Property = require('./property').Property,
+    E = '',
+    DEFAULT_SSH_PORT = 22,
+
+    SshAuth;
+
+/**
+* The following is the object structure accepted as constructor parameter while calling `new SshAuth(...)`. It is
+* also the structure exported when {@link Property#toJSON}.
+* @typedef SshAuth~definition
+*
+* @property {String=} host
+* @property {Number=} port
+* @property {String=} username
+* @property {String=} password
+* @property {String=} privateKeyPath
+* @property {String=} passPhrase
+*
+* @example <caption>JSON definition of an example SshAuth object</caption>
+* {
+*       "host": "54.111.103.121",
+*       "port": "22"
+*       "username": "someuser",
+*       "password": "somepassword",
+*       "privateKeyPath": "/path/to/key/file",
+*       "passPhrase": "somepassphrase"
+* }
+*/
+_.inherit((
+
+    /**
+     * A SSH Auth definition which represents the auth configuration requried to establish a ssh connection
+     *
+     * @constructor
+     * @extends {Property}
+     * @param {SshAuth~definition=} [definition]
+     *
+     * @example <caption>Create a new SshAuth</caption>
+     * var SshAuth = require('postman-collection').SshAuth,
+     *     mySshAuth = new SshAuth({
+     *          host: "54.111.103.121",
+     *          port: "22"
+     *          username: "someuser",
+     *          password: "somepassword",
+     *          privateKeyPath: "/path/to/key/file",
+     *          passPhrase: "somepassphrase"
+     *      });
+     */
+    SshAuth = function SshAuth (definition) {
```

Comment:
`SSHAuth`

Reply:
Actually, there's a test in postman-collection which prevents this. Basically, test enforces that first character should be Uppercase and rest lowercase.